README.md

```diff
@@ -51,73 +117,39 @@ There are multiple options to run Docker locally. One such option is [colima](ht
 ```
 brew install docker
 brew install colima
-colima start
 ln -sf ~/.colima/docker.sock /var/run/docker.sock
 ```
 
 The `ln` at the end is because Supabase requires interacting with the local Docker API. See [this Supabase issue](https://github.com/supabase/cli/issues/153) and [this colima issue.](https://github.com/abiosoft/colima/issues/144) This likely requires `sudo`.
 
+### Installation
 
-**Github App**
+Make sure the repo is cloned, and you're working in the root directory of the repo:
 
-The `draft` web front end uses Github for authentication. In order to login locally, you'll need to create a new application for development. Visit [https://github.com/settings/apps/new](https://github.com/settings/apps/new) to create a new Github app using the following parameters:
-1. *name*: draft-local-dev (recommended, but not required)
-2. *Homepage URL:* http://localhost:54321
-3. *Callback URL:* http://localhost:54321/auth/v1/callback
-4. *Request user authorization (OAuth) during installation:* yes
-5. *Webhook active:* false
-6. *Permissions:* Read-only access to email address
-7. *Where can this GitHub App be installed?:* Only on this account
-
-Once you have created the app, you'll need to create a file `server/.env` and add both the `CLIENT_ID` and a generated `CLIENT_SECRET`.
-
-```
-SUPABASE_AUTH_GITHUB_CLIENT_ID="<CLIENT_ID>"
-SUPABASE_AUTH_GITHUB_SECRET="<CLIENT_SECRET>"
 ```
-
-
-**Supabase CLI**
-
-```
-brew install supabase/tap/supabase
+git clone https://github.com/eriktaubeneck/draft.git
```

Comment:
this seems to imply for users to work off their own fork, is it required or they can simply clone 
`private-attribution/draft`?

Reply:
good catch, this hasn't been updated since I moved it.